README.md

```diff
@@ -22,6 +22,15 @@ curl https://cdn.rawgit.com/pimterry/git-confirm/v0.2.1/hook.sh > .git/hooks/pre
 ```
 (*Note the version number*)
 
+or to install it globally on v2.9+, you can run:
+
+```bash
+mkdir ~/.git-hooks
+git config --global core.hooksPath ~/.git-hooks
```

Comment:
After using this for a few days, I realize this line actually doesn't do what I thought it did. I thought this would _add_ a hook folder to the per-repo one, so that I could have some global hooks and some local ones. Instead, this replaces it. So now none of my per-repo hooks work anymore. For that reason, I'm going to close this PR

Reply:
Fair enough, that makes sense! There is the possibility to set `init.templatedir` like so:

```bash
git config --global init.templatedir '~/.git-hooks-template'
```

that would let you set hooks up automatically in new repos, whilst still allowing per-repo hooks to be added. It doesn't help with existing repos though unless you go and run `git init` in each to update every one. I think just doing it on a per-repo basis is probably fine most of the time.